rust-runtime/aws-smithy-http/src/operation.rs

```diff
@@ -9,11 +9,17 @@
 use aws_smithy_types::config_bag::{Storable, StoreReplace};
 use std::borrow::Cow;
 
-//TODO(runtimeCratesVersioningCleanup): Re-point those who use the following reexport to
-// directly depend on `aws_smithy_types` and remove the reexport below.
+//TODO(runtimeCratesVersioningCleanup): Re-point those who use the deprecated type aliases to
```

Comment:
Is this TODO for pre-GA or will we do this post GA?

Reply:
I'd expect it's for pre-GA. After we've released a new version (likely a breaking release), we'll go ahead and remove those deprecated type aliases.